### DISCUSSION TITLE
Apache reverse proxy

### DISCUSSION THREAD

**Discussion by adamaze** [kubero-dev/ladder#66] [Q&A]
I'll admit I'm not a pro with Apache reverse proxies, but I have figured out how to get them to work for a bunch of other containers I run at home. I set up a ladder container in my standard setup so I can use https://ladder.mydomain.com, and it loads up just fine, but whenever I try to use it, I get a message like this:

Get "https:/CNN.com": http: no Host in request URL

If I access it without the reverse proxy it works fine. Is there something I need to set to get it working?

**Comment by bgro82** [comment]
I have run into this exact same issue. Were you able to figure this out @adamaze?

**Reply by adamaze** [reply]
Sadly, no. Sorry!

**Comment by gitnew12** [comment]
I am using Traefik v3 reverse proxy and just ran into this issue. Is there an environment variable in the docker compose that would handle this?
